flashchips.c

```diff
@@ -3441,6 +3441,45 @@ const struct flashchip flashchips[] = {
 		.read		= read_memmapped,
 		.voltage	= {3000, 3600},
 	},
+	
+    	{
+		.vendor		= "BOYAMICRO",
```

Comment:
How about "Boya" since the company refers to itself as "Boya Microelectronics" in the datasheet?

Reply:
Updated